### PR TITLE
feat: support expiring applications

### PR DESCRIPTION
**Pull request by jleon15**
## Description

- Support expiring applications by adding expires_at and can_create_expiring_applications flag. 

## Testing required outside of automated testing?

- [x] Not Applicable

### Screenshots (if appropriate):

- [x] Not Applicable

## Rollback / Rollforward Procedure

- [x] Roll Forward
- [ ] Roll Back

## Reviewer Checklist

- [ ] Description of Change
- [ ] Description of outside testing if applicable.
- [ ] Description of Roll Forward / Backward Procedure
- [ ] Documentation updated for Change

### CODE REVIEW

File: BasisTheory/APIs.swift [hunk 1]
```diff
@@ -14,7 +14,7 @@ import Foundation
 public typealias BasisTheory = BasisTheoryAPI
 
 open class BasisTheoryAPI {
-    public static var basePath = "https://api.basistheory.com"
+    public static var basePath = "https://api-dev.basistheory.com"
```
Comment (kevinperaza) [inline BasisTheory/APIs.swift:17]: I don't think this should target dev?

File: BasisTheory.podspec [hunk 2]
```diff
@@ -4,8 +4,8 @@ Pod::Spec.new do |s|
   s.osx.deployment_target = '10.11'
   s.tvos.deployment_target = '9.0'
   s.watchos.deployment_target = '3.0'
-  s.version = '0.3.0'
-  s.source = { :git => 'git@github.com:Basis-Theory/basistheory-swift.git', :tag => '0.3.0' }
+  s.version = '0.0.1'
+  s.source = { :git => 'git@github.com:Basis-Theory/basistheory-swift.git', :tag => '0.0.1' }
```
Comment (brigonzalez) [inline BasisTheory.podspec:8]: this needs to be reverted

File: BasisTheory/APIs.swift (hunk 1 above)
Comment (brigonzalez) [inline BasisTheory/APIs.swift:17]: default should be prod

File: .openapi-generator/VERSION [hunk 3]
```diff
@@ -1 +1 @@
-6.2.1-SNAPSHOT
\ No newline at end of file
+6.3.0-SNAPSHOT
```
Comment (dhudec) [inline .openapi-generator/VERSION:1]: We need to pin this version in the script too. I will do this on Monday, don't worry about it and enjoy the time off.

**Review by dhudec — CHANGES_REQUESTED** [review]

**Review by dhudec — APPROVED** [review]

**Review by kevinperaza — APPROVED** [review]

**Pull request merged**